requirements.txt

```diff
@@ -1,2 +1,19 @@
-elasticsearch-curator==5.4.0
```

Comment:
Updated `search-curator` to 5.8.1 which supports all Elasticsearch versions 5.x through 7.x. See https://www.elastic.co/guide/en/elasticsearch/client/curator/5.8/version-compatibility.html

Also frozen all pip requirements.